src/wizard-dialog.ts

```diff
@@ -76,6 +78,7 @@ export class WizardDialog extends LitElement {
 
   /** Commits `action` if all inputs are valid, reports validity otherwise. */
   async act(action?: WizardAction): Promise<void> {
+    if (this.actionCalled) return;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7195000)
```suggestion
    if (this.actionCalled) {return;}
```